docs/using/java.rst

```diff
@@ -32,15 +32,15 @@ To create an app, you use the command `app create`:
 
 ::
 
-    $ tsuru app create <app-name> <app-platform>
+    $ tsuru app create <app-name> <app-platform> <team>
 
 For Java, the app platform is, guess what, ``java``! Let's call our application "helloweb":
 
 .. highlight:: bash
 
 ::
 
-    $ tsuru app create helloweb java
+    $ tsuru app create helloweb java --team
```

Comment:
this is not accurate, if the user runs this exact command tsuru will ask the user to specify a valid team